logging/dedicated-efk-logging.adoc

```diff
@@ -0,0 +1,65 @@
+:context: dedicated-efk-logging
+[id="dedicated-efk-logging"]
+= About cluster logging and {product-title}
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+As an {product-title} cluster administrator, you can deploy cluster logging
+to aggregate logs for a range of services.
+
+{product-title} clusters come configured with logging via `ElasticSearch`
+operators installed and `Curator` pre-installed to have a two-day retention of
```

Comment:
Curator is _pre-configured_ for 2 day retention per https://jira.coreos.com/browse/SREP-1733

Reply:
Simplified it to:
" {product-title} clusters can perform logging tasks via ElasticSearch
and Curator Operators that are pre-configured to have a two-day retention of
logs."

Reply:
Curator isn't technically an Operator. I call it a _tool_ in the logging docs, but only twice. Something like this? 
_{product-title} clusters can perform cluster logging using the Elasticsearch`
Operator, which is pre-installed. Cluster logging is configured through the `Curator` tool to retain logs for two days.

Reply:
Yep, that's perfect. Thanks!